components/components-styles/alert.css

```diff
@@ -0,0 +1,55 @@
+:root{
+    --primary-alert-color:#1950a1;
+    --secondary-alert-color:#55595e;
+    --success-alert-color:#1e5c3f;
+    --danger-alert-color:#973e46;
+    --warning-alert-color:#a18d51;
+    --info-alert-color:#145d6c;
+    --primary-alert-bgcolor:#cfe2ff;
+    --secondary-alert-bgcolor:#e2e3e5;
+    --success-alert-bgcolor:#d1e7dd;
+    --danger-alert-bgcolor:#f8d7da;
+    --warning-alert-bgcolor:#fff3cd;
+    --info-alert-bgcolor:#cff4fc;
+}
```

Comment:
Change the color names to the capital letters Would become easy for you later on to read and react accordingly.
though great UI

Reply:
Else everything looks amazing Keep up the good work. :)

Reply:
Okay. Will change that.

Reply:
> Change the color names to the capital letters Would become easy for you later on to read and react accordingly. though great UI

Hi, Akshat I have made the above changes please do a review again.

Reply:
LGMT.

Reply:
Make sure the contrast ratio is good. It's a bit low for warning alert.